doc/release/README.prereqs

```diff
@@ -24,3 +24,15 @@ about your environment for using Chapel:
   code using compilers by Cray, IBM, Intel, and PGI. Any standard C and
   C++ compiler should work, but the aforementioned are the ones we have
   the most experience with.  
+
+
+We have used the following commands to set up newly installed system images.
+Note that our test system works much better with tcsh installed and that users
+of Chapel won't need 'git'.:
+
+CentOS 6.5, 6.6, 7.0, SLES 11 SP3:
+sudo yum install git gcc gcc-c++ bc perl python tcsh bash
```

Comment:
I don't think git should be included. Developers may need other things like flex, bison, gcc 4.7+, or python 2.7 depending on what they are working on. It seems unfortunate to complicate the user requirements with the developer requirements (which could go in a doc under doc/developer/).